src/services/ui/src/pages/form/index.tsx

```diff
@@ -26,80 +26,25 @@ export function Webform() {
     id: string;
     version: string;
   }>();
-  console.log({ id, version });
-
   const defaultValues = documentInitializer(ABP1);
 
+  const savedData = localStorage.getItem(`${id}v${version}`);
```

Comment:
Question about process, can a state have more than one of the same form type drafting at a time?